src/types.js

```diff
@@ -0,0 +1,14 @@
+import { string, shape } from "prop-types";
```

Comment:
This is a centralized propType. [Here's a blog post that explains why](https://medium.freecodecamp.org/react-pattern-centralized-proptypes-f981ff672f3b).